src/core/tests/aio.cpp

```diff
@@ -34,6 +34,7 @@
  */
 
 
+# include <future>
```

Comment:
fixed in https://github.com/qinzuoyan/rDSN/commit/6f6bdfa1b939185f4e55a6252d34e7e092878f8c